apps/phi-das/src/config.ts

```diff
@@ -1,12 +1,13 @@
 import * as dotenv from 'dotenv';
 
-export type AppConfig = {
-	port: string;
-};
-
-export const getAppConfig = (): AppConfig => {
+export const getAppConfig = () => {
 	dotenv.config();
 	return {
-		port: process.env.PORT || '8083',
+		isProduction: process.env.NODE_ENV === 'production',
+		express: {
+			port: process.env.PORT || '8081',
```

Comment:
PHI Das defaults to port `8083`
```suggestion
			port: process.env.PORT || '8083',
```